app/src/test/java/com/codesoom/assignment/AppTest.java

```diff
@@ -0,0 +1,41 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.controllers.HomeController;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(HomeController.class)
+@DisplayName("APP 클래스")
+class AppTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Nested
+    @DisplayName("checkServerHealth(HomeController) 메소드는")
+    class Describe_checkServerHealth {
+        @Test
+        @DisplayName("호출하면 서버 상태를 리턴한다")
+        void it_returns_server_status() throws Exception {
+            mockMvc.perform(get("/healthcheck"))
+                    .andExpect(status().isOk())
+                    .andExpect(content().string("Server is OK"));
+        }
```

Comment:
이렇게 하면 어떨까요? 항상 좀 더 자연스럽고 부드러운 표현이 없을지 고민해 보세요. 조금씩만 더 유연하게 생각해보시면 됩니다.

```suggestion
    @Nested
    @DisplayName("헬스 체크를 하면")
    class Describe_checkServerHealth {
        @Test
        @DisplayName("서버가 살아있다고 응답한다.")
        void it_returns_server_status() throws Exception {
            mockMvc.perform(get("/healthcheck"))
                    .andExpect(status().isOk())
                    .andExpect(content().string("Server is OK"));
        }
```

Reply:
네 감사합니다~!!